modules/gitops-argo-cd-properties.adoc

```diff
@@ -108,6 +146,8 @@ a|* `<autotls>` - Use the provider to create the Redis server's TLS certificate.
 
 |`resourceInclusions` |The configuration to identify which resource group/kinds are applied.|`__<empty>__` |
 
+|`ResourceTrackingMethod` |The field used by Argo CD to monitor of its managed resources.|`__<label>__` |
+
 |`server` |Argo CD Server configuration options.|`__<object>__`
 a|* `<autoscale>` - Server autoscale configuration options.
   * `<extraCommandArgs>` - List of arguments added to the existing arguments set by the Operator.
```

Comment:
Again, for the lines below where I cannot add comments:
from:
  * `<host>` - The hostname used for Ingress or Route resources.
  to:
  * `<host>` - The hostname used for `Ingress` or `Route` resources.

Reply:
Fixed.